workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -156,6 +156,7 @@ PlayerSettings:
   applicationIdentifier:
     Android: io.improbable.gdk
     Standalone: com.improbable.gdk.starterproject
+    iOS: com.improbable.gdk.jon
```

Comment:
why not .starterproject?

Reply:
Darnit I thought I'd removed my name already haha. `com.improbable.starterproject`, then?